aptos-move/framework/aptos-framework/sources/aptos_coin.spec.move

```diff
@@ -15,6 +15,8 @@ spec aptos_framework::aptos_coin {
         aborts_if exists<coin::CoinInfo<AptosCoin>>(addr);
         aborts_if !exists<aggregator_factory::AggregatorFactory>(addr);
         ensures exists<MintCapStore>(addr);
+        // property 3: The abilities to mint Aptos tokens should be transferable, duplicatable, and destroyable.
+        ensures global<MintCapStore>(addr).mint_cap ==  MintCapability<AptosCoin> {};
```

Comment:
Could you remove this `ensures` line? I think `ensures exists<MintCapStore>(addr);` is the right spec for the high-level requirement.

Reply:
I think it could be remain as an local specification?